src/pages/settings.tsx

```diff
@@ -1,45 +1,87 @@
-import { Form, Input, Button, Tabs ,Layout , Typography, Space} from "antd";
+import { Form, Input, Button, Tabs, Layout, Typography, Space, Card, Col, Row } from "antd";
 import LayoutMaster from "../hoc/layout";
 import ChangePassword from "../components/ComponentPages/ChangePassword"
-
+import Login from "../components/componentPages/Login";
+import GetStartedLayout from "../hoc/getStartedLayout";
+import useAuthentication from "../hooks/isUserAuthenticated";
+import { useSelector } from "react-redux";
+import { useState } from "react";
+import { RootState } from "../store";
 const { TabPane } = Tabs;
 const { Header, Footer, Sider, Content } = Layout;
 const { Title, Text } = Typography;
+const tabList = [
+    {
+        key: 'profileinfo',
+        tab: 'Profile Info',
+    },
+    {
+        key: 'changepassword',
+        tab: 'Change Password',
+    },
+];
+
+const contentList = {
+    profileinfo: <p> Profile Info</p>,
+    changepassword: <p><ChangePassword></ChangePassword></p>,
+};
 //Route : /settings
 export default function Settings() {
- 
-  const OperationsSlot = {
-    left: <Space><Text></Text><Title level={4} >
-    Account Settings
-  </Title><Text></Text> </Space>
-  };
-  return (
-    <>
-    <LayoutMaster>
-        <Layout>
-        <Layout>
-            <Sider style={{
-            background:'#3ba0e9'
-        }}>Advertisement</Sider>
-            <Content>
-                <div className="card-container">
-                    <Tabs  tabBarExtraContent={OperationsSlot}>
-                        <TabPane tab="Profile Info" key="1">
-                            Profile Info
-                        </TabPane>
-                        <TabPane tab="Change Password" key="2">
-                            <ChangePassword></ChangePassword>
-                        </TabPane>
-                    </Tabs>
-                </div>
-            </Content>
-            <Sider
-             style={{
-                background:'#3ba0e9'
-            }}>Advertisement</Sider>
-        </Layout>
-        </Layout>
-    </LayoutMaster>
-    </>
-  );
+
+    const [activeTabKey, setActiveTabKey] = useState('profileinfo');
+
+    const onTab1Change = key => {
+        setActiveTabKey(key);
+    };
+    const { isUserAuthenticated } = useAuthentication();
+    const isAuthenticate = useSelector(
+        (state: RootState) => state.auth.authenticated
+    );
+
+    return (
+        <>
+            {isAuthenticate ?
+                <LayoutMaster>
+                    <Layout>
```

Comment:
Can we move this layout to a separate component and reduce the size of this component? As we are using the `ternary` operator, I think we should move the result in a new component and render this in case of the true condition

Reply:
Done